src/audio/kpb.c

```diff
@@ -70,6 +70,7 @@ static void kpb_buffer_data(struct comp_data *kpb, struct comp_buffer *source,
 			    size_t size);
 static void kpb_allocate_history_buffer(struct comp_data *kpb);
 static void kpb_clear_history_buffer(struct hb *buff);
+static int kpb_is_history_buffer_big_enough(struct hb *buff);
```

Comment:
Well, the idea to move it out of allocation function was to make it more modular, so for example if we add an option to dynamically resize history buffer we can quickly check it with this function. As for now I agree, it can be verified right after allocation and that's how I updated it.